src/murmur/Murmur.ice

```diff
@@ -823,6 +823,11 @@ module Murmur
 		  * @returns An ID-list of users listening to the given channel
 		  */
 		 idempotent IntList getListeningUsers(int channelid);
+
+		 /**
+		  * @param receiverUserIDs list of IDs of the users the message shall be sent to
+		  */
+		 idempotent void sendWelcomeMessage(IdList receiverUserIDs);
```

Comment:
Yes exactly. This is just to give more control over the behavior. I don't see a use case where one only wants to broadcast it to a subset of users, but providing the option seems to be better than anticipating that this will never be needed.
And given that it is quite easy to obtain a list of all connected users, I think this doesn't really hurt :shrug: